Client/Frontend/Home/Wallpapers/WallpaperDataManager.swift

```diff
@@ -112,22 +123,31 @@ struct WallpaperDataManager {
     }
     
     private func firefoxDefaultCollection() -> [WallpaperCollection] {
-        return [WallpaperCollection(wallpaperFileNames: ["fxCerulean",
-                                                         "fxAmethyst",
-                                                         "fxSunrise"],
-                                    ofType: .themed(type: .firefox))]
+        return [WallpaperCollection(
+            wallpaperFileNames: [WallpaperID(name: "fxSunrise",
+                                             accessibilityID: accessibilityIDs.FxSunriseWallpaper)],
+            ofType: .themed(type: .firefox)),
+                WallpaperCollection(
+            wallpaperFileNames: [WallpaperID(name: "fxCerulean",
+                                             accessibilityID: accessibilityIDs.FxCeruleanWallpaper),
+                                 WallpaperID(name: "fxAmethyst",
+                                             accessibilityID: accessibilityIDs.FxAmethystWallpaper)],
```

Comment:
Looked at the wallpaper strings. It also contains the same issue with the `, button` at the end. They'll need to be adjusted 🤔